### PR TITLE
A handful of small updates

### PR DESCRIPTION
**Pull request by ajs256**
- some clarifications in `changed-options.md`
- clarifications and grammar fixes in `chat-reporting-faq.md`
- link to official CC0 site in `language-support.md`
- clarifications to `vanilla-launcher-faq.md`

### CODE REVIEW

File: language-support.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Language support
 
-You can translate the mods themselves and the modpack-specific content. All modpack-specific translations are licensed under [CC0](https://www.tldrlegal.com/license/creative-commons-cc0-1-0-universal), meaning they can be freely copied, modified, republished and so on.
+You can translate the mods themselves and the modpack-specific content. All modpack-specific translations are licensed under [CC0](https://creativecommons.org/publicdomain/zero/1.0/), meaning they can be freely copied, modified, republished and so on.
```
Comment (Kichura) [inline language-support.md:3]: This should not be changed as certain people need simpler information to understand licenses properly in this context.

File: changed-options.md [hunk 2]
```diff
@@ -8,17 +8,17 @@ Because the pack is using YOSBR, the options will only change if you do not have
 
 ### Options
 
-| Option | Description | Vanilla | Modpack | Reason for Change |
+| Option | Description | Vanilla default | Modpack | Reason for Change |
```
Comment (Kichura) [inline changed-options.md:11]: Revert this.

**Review by Kichura — CHANGES_REQUESTED** [review]

File: changed-options.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ Because the pack is using YOSBR, the options will only change if you do not have
 
 ### Options
 
-| Option | Description | Vanilla default | Modpack | Reason for Change |
+| Option | Description | Vanilla | Modpack | Reason for Change |
 | - | - | - | - | - |
 | advancedItemTooltips | Enables ["advanced" tooltip info](https://www.online-tech-tips.com/wp-content/uploads/2021/01/Armor-Tooltips-610x571.png) on items - item ID and durability, and exact color hex code for dyed armor | false | true | These are useful for much more than just debugging - if you're using commands, want to know the exact durability value, learning English, etc. Plus, this setting is hidden in the debug menu, so many people might not know about it.. |
```
Comment (Kichura) [inline changed-options.md:13]: there is a double period at the end of line 13.

**Comment by Madis0** [comment]
Thank you!

**Pull request merged**